README.md

```diff
@@ -273,36 +273,31 @@ And here it is: a resource pool called `common-resource-pool` that is using the
 
 Now let's see how a network is defined:
 
-```
+```yaml
 networks:
 - name: webapp-network
   type: manual
   subnets:
-  - range: 10.244.0.0/30
-    gateway: 10.244.0.1
+  - range: 10.244.1.0/24
+    gateway: 10.244.1.1
     static:
-      - 10.244.0.2
-    cloud_properties:
-      name: random
-  - range: 10.244.0.4/30
-    gateway: 10.244.0.5
-    static: []
+      - 10.244.1.2
     cloud_properties:
       name: random
 ```
 
-Now that's scary! Well, not as much - it's more verbose than complex. What we are doing is the following: 
+Now that's scary! Well, not as much - it's more verbose than complex. What we are doing is the following:
 - we declare there is one network called `webapp-network` and it has type `manual`. This means that we will define all subnets by hand.
-- we define several subnets. The rule we are following is that we have a separate subnet for every ip that we need in the deployment. Every subnet that we are defining has a network mask of 255.255.255.252 or 30 bits. This leaves only 4 addresses as follows (for example):
-  - 10.244.0.0 - this is the network address
-  - 10.244.0.1 - this is normally used as the gateway
-  - 10.244.0.2 - this is an ip address that bosh can use for something
-  - 10.244.0.3 - this is a broadcast address
-In the first subnet we are using 10.244.0.1 as the gateway and we tell bosh that we want one static ip (10.244.0.2). We assigned this static ip to the job, because we want it to be available on the same address on every start. In the second subnet we did not reserve any static ips which means that bosh will dynamically assign the ip 10.244.0.6 to whatever machine needs an ip. In our case this will be used for a bosh worker vm that does package compilation.
+- we define a subnet. The subnet that we are defining has a network mask of 255.255.255.0 or 24 bits. This leaves only 256 addresses as follows (for example):
+  - 10.244.1.0 - this is the network address
+  - 10.244.1.1 - this is normally used as the gateway
+  - 10.244.1.2, 10.244.1.64 - those are ip addresses that bosh can use for something
```

Comment:
I meant that both 2 and 64 are IP addresses that BOSH can use. Maybe
`10.244.1.2, ..., 10.244.1.254` makes things more clear?